definition_validator/validator.cc

```diff
@@ -190,6 +192,27 @@ class ValidateWalk {
 private:
     UnorderedMap<core::SymbolRef, vector<core::SymbolRef>> abstractCache;
 
+    struct GenericFrame {
+        bool isStdlib = false;
```

Comment:
Could you please leave a comment on why it matters if it comes from stdlib or not?